static/js/SearchSheetResult.jsx

```diff
@@ -12,72 +12,77 @@ import {
 
 class SearchSheetResult extends Component {
     handleSheetClick(e) {
-      var s = this.props.data._source;
+      const s = this.props.metadata;
       if (this.props.onResultClick) {
         e.preventDefault()
         this.props.onResultClick("Sheet " + s.sheetId);
       }
       Sefaria.track.event("Search", "Search Result Sheet Click", `${this.props.query} - ${s.sheetId}`);
     }
     handleProfileClick(e) {
-      const s = this.props.data._source;
+      const s = this.props.metadata;
       Sefaria.track.event("Search", "Search Result Sheet Owner Click", `${this.props.query} - ${s.sheetId} - ${s.owner_name}`);
     }
-    get_snippet_markup(data) {
-      let snippet = data.highlight.content.join("..."); // data.highlight ? data.highlight.content.join("...") : s.content;
-      snippet = snippet.replace(/^[ .,;:!-)\]]+/, "");
+    get_snippet_markup() {
+      const snippet = this.props.snippet.replace(/^[ .,;:!-)\]]+/, "");
       const lang = Sefaria.hebrew.isHebrew(snippet) ? "he" : "en";
       return { markup: {__html: snippet}, lang };
     }
+    formatDate(dateString) {
+        const date = new Date(dateString);
```

Comment:
Is this code from GPT? I'd never seen this class before but it looks very helpful. I would remove the comments since they basically just say what is already obvious from the code